tracing-bom/build.gradle

```diff
@@ -3,13 +3,20 @@ plugins {
 }
 
 micronautBom {
-     extraExcludedProjects.add 'kotlin-tests'
-     suppressions {
-            bomAuthorizedGroupIds.put(
+    extraExcludedProjects.add 'kotlin-tests'
+    suppressions {
+        bomAuthorizedGroupIds.put(
                 "io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha",
-                ["io.opentelemetry.javaagent", "io.opentelemetry", "io.opentelemetry.instrumentation", "io.opentelemetry.javaagent.instrumentation"] as Set
-            )
-            dependencies.add("io.opentelemetry:opentelemetry-bom:1.15.0")
-            dependencies.add("io.opentelemetry:opentelemetry-bom-alpha:1.15.0-alpha")
-     }
+                [
+                        "io.opentelemetry",
+                        "io.opentelemetry.instrumentation",
+                        "io.opentelemetry.javaagent",
+                        "io.opentelemetry.javaagent.instrumentation"
+                ] as Set
+        )
+        dependencies.add("io.opentelemetry:opentelemetry-bom:1.18.0")
+        dependencies.add("io.opentelemetry:opentelemetry-bom:1.19.0")
```

Comment:
@n0tl3ss No, I'm not sure that I did everything correctly in the bom file. I think I had to add version 1.18.0 because of this line:
```
managed-opentelemetry-instrumentation-alpha = '1.18.0-alpha'
```
But, as I see it, you have already changed the version to 1.19.0-alpha and fixed the bom module. So it should be alright by now